src/main/java/eu/dissco/backend/controller/BaseController.java

```diff
@@ -41,4 +42,9 @@ protected List<String> getMassFromRequest(JsonApiRequestWrapper requestBody)
     return Arrays.asList(
         mapper.treeToValue(requestBody.data().attributes().get("mass"), String[].class));
   }
+
+  protected String getNameFromToken(Authentication authentication) {
+    return authentication.getName();
```

Comment:
Not sure if this method is still needed. Previously it was more difficult to get the token but now it is shorter than calling this method...